pkg/collectors/subsystem.go

```diff
@@ -245,35 +244,61 @@ func (f *PerfCollector) collectSystemMetrics(ch chan<- prometheus.Metric, fsRest
 	return true
 }
 
-func (f *PerfCollector) createSystemPhysicalCapacityMetrics(ch chan<- prometheus.Metric, sysInfoResults rest.StorageSystem, systemName SystemName) {
-
+func (f *PerfCollector) createSystemPhysicalCapacityMetrics(ch chan<- prometheus.Metric, sysInfoResults rest.StorageSystem, systemName SystemName, fsRestClient *rest.FSRestClient) {
 	// [lssystem]: physical_capacity
-	physicalTotalCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalTotalCapacity].(string), 64)
+	physicalTotalCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalTotalCapacityKey].(string), 64)
 	if err != nil {
-		log.Errorf("get physical total capacity failed: %s", err)
+		log.Errorf("get system physical total capacity failed: %s", err)
 		return
 	}
 	// [lssystem]: physical_free_capacity
-	physicalUsableCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalFreeCapacity].(string), 64)
+	physicalUsableCapacity, err := strconv.ParseFloat(sysInfoResults[PhysicalFreeCapacityKey].(string), 64)
 	if err != nil {
-		log.Errorf("get physical usable capacity failed: %s", err)
+		log.Errorf("get system physical usable capacity failed: %s", err)
 		return
 	}
-	physicalReclaimableCapacity, err := strconv.ParseFloat(sysInfoResults[ReclaimableCapacity].(string), 64)
+	physicalReclaimableCapacity, err := f.calcSystemReclaimableCapacity(fsRestClient)
+	//physicalReclaimableCapacity, err := strconv.ParseFloat(sysInfoResults[ReclaimableCapacityKey].(string), 64)
 	if err != nil {
-		log.Errorf("get physical reclaimable capacity failed: %s", err)
+		log.Errorf("get system physical reclaimable capacity failed: %s", err)
 		return
 	}
-	physicalFreeCapacity := physicalUsableCapacity + physicalReclaimableCapacity
+	physicalUsedCapacity := physicalTotalCapacity - physicalUsableCapacity - physicalReclaimableCapacity
+
+	physicalFreeCapacity := physicalTotalCapacity - physicalUsedCapacity
 	// used = total - free
-	physicalUsedCapacity := physicalTotalCapacity - physicalFreeCapacity
 	log.Infof("system capacity total: %f, free: %f, used: %f", physicalTotalCapacity, physicalFreeCapacity, physicalUsedCapacity)
 
 	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalTotalCapacity], physicalTotalCapacity, &systemName)
 	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalUsedCapacity], physicalUsedCapacity, &systemName)
 	newSystemCapacityMetrics(ch, f.sysCapacityDescriptors[SystemPhysicalFreeCapacity], physicalFreeCapacity, &systemName)
 }
 
+func (f *PerfCollector) calcSystemReclaimableCapacity(fsRestClient *rest.FSRestClient) (float64, error) {
+	var ReclaimableSum float64
```

Comment:
consider changing to `totalSystemReclaimable` or `totalReclaimable`

Reply:
fixed.